app/src/main/res/raw-bg/help_problems.html

```diff
@@ -5,9 +5,8 @@
 <html>
 <head><title>Общи проблеми</title></head>
 <body>
-<h2>Copy of hosts file failed on Android 9+</h2>
-<p>Latest versions of Android use read-only <code>/system</code> partition.<br>
-If you are using Magisk as root solution, ensure you enable the systemless hosts module then reboot.</p>
+<h2>Копиране на хост файла се провали на Android 9+</h2>
+<p>Последните версии на </p>
```

Comment:
trasifex does not update the branch with newer versions of the translation.
is already fixed in transifex